src/DynamoCoreWpf/ViewModels/Preview/WatchViewModel.cs

```diff
@@ -31,6 +33,19 @@ internal void Click()
         private string _path = "";
         private bool _isOneRowContent;
         private readonly Action<string> tagGeometry;
+        private bool _isCollection;
+
+        // Instance variable for the number of items in the list 
+        private int numberOfItems;
+
+        // Instance variable for the max depth of items in the list
+        private int maxListLevel;
+
+        //Instance variable for the current list depth of items in the list
+        private int listDepth;
+
+        //Instance variable for list of list level items
+        private List<ListLevel> _listlevelsList;
```

Comment:
naming: remove '_' before '_listlevelsList'. Rename '_listlevelsLists' to 'levels'